### ISSUE TITLE
The ACME-UNIFIED environment is going to change its name.

### ISSUE THREAD

**Issue by sterlingbaldwin** [ACME-Climate/a-prime#12]
I dont know what the timeline is, but the ACME-UNIFIED environment will at some point be changing its name to E3SM-UNIFIED. The lines that reference the files are going to need to change

https://github.com/ACME-Climate/a-prime/blob/master/run_aprime.bash#L359

**Comment by xylar** [comment]
Indeed, as I'll be the one updating the acme-unified environments on each of the supported machines, I'll make sure to coordinate with A-Prime to keep them up-to-date.  The cleanest way to handle the change will be for me to create new environments with the new name but leave the old ones in place until everyone (primarily A-Prime) has made the transition.

Thanks, @sterlingbaldwin for keeping an eye on this.
